prql-compiler/src/lib.rs

```diff
@@ -40,6 +40,11 @@ pub fn to_json(prql: &str) -> Result<String> {
     Ok(serde_json::to_string(&parse(prql)?)?)
 }
 
+// Convert JSON AST back to PRQL string
+pub fn from_json(json: &str) -> Result<String> {
```

Comment:
Possibly these should be returning a `Result<Query>`, similar for `to_json`. But this is fine for now